src/components/forms/formr-part-b/viewSections/ViewSection1.cy-test.tsx

```diff
@@ -0,0 +1,56 @@
+import { mount } from "@cypress/react";
+import { Provider } from "react-redux";
+import { Router } from "react-router-dom";
+import { submittedFormRPartBs } from "../../../../mock-data/submitted-formr-partb";
+import store from "../../../../redux/store/store";
+import ViewSection1 from "./ViewSection1";
+import history from "../../../navigation/history";
+import ViewSectionShouldIncludeThisData, {
+  ISectionDataField
+} from "./ViewSectionTestHelper";
+
+const makeSectionEditButton = (section: number) => {
+  return false;
```

Comment:
@john12321 this was intended as a bit of a placeholder function, but it did mean I had to change the signature of the IViewSection[x] from returning `false | JSX.Element` to  `boolean | JSX.Element`.
Not sure if (a) this is appropriate and (b) we might need tests for this

Reply:
Hi @ReubenRobertsHEE , just pulled the branch to have a look. Re above, makes sense changing to boolean type. Will have a closer look at this and tests etc.. 
One obvious bit of refactoring I noticed in this file (and other files) is to define the `formData` and `viewSectionProps` vars within the describe block and also use  `beforeEach` inside describe to mount `ViewSection1` .

Reply:
Done, thanks @john12321